js/index.js

```diff
@@ -1 +1,133 @@
 // Your code goes here
+//Mouse Over / changes anchor colors
+const navAnchors = document.querySelector('.nav');
+
+navAnchors.addEventListener('mouseover', () => {
+    navAnchors.style.backgroundColor = 'gold';
+    console.log('gold nav bar')
+    
+});
+
+navAnchors.addEventListener('mouseleave', () => {
+    navAnchors.style.backgroundColor = 'white';
+    console.log('default nav bar')
+});
+
+navAnchors.addEventListener('click', (event) => {
+    event.preventDefault();
+});
+
+
+//Key Down / Press any key for a secret message for any plagerizers
+const pageBody = document.querySelector('body');
+const noPlagerismFont = document.querySelector('.footer p');
+
+pageBody.addEventListener('keydown', () => {
+    noPlagerismFont.textContent = 'Copyright Fun Bus 2019. https://www.copyright.gov/help/faq/definitions.html';
+    console.log('alerted plagerizers')
+
+});
+
+//Wheel / make font size smaller for h1 
+const title = document.querySelector('h1');
+
+pageBody.addEventListener('wheel', () => {
+    title.style.fontSize = '22px';
+
+    console.log('font size changed')
+
+});
+
+//Drag & Drop / drags the second image
+const dragImage = document.querySelector('.content-section img')
+
+dragImage.addEventListener('drag', (event) => {
+    console.log('dragging')
+
+    event.stopPropagation();
+
+}, false);
+
+dragImage.addEventListener("dragstart", () => {
+    dragged = event.target;
+    event.target.style.opacity = .1;
+
+    console.log('drag start')
+
+
+}, false);
+
+dragImage.addEventListener("dragend", () => {
+    event.target.style.opacity = '3%';
+
+    console.log('drag end')
+
+
+}, false);
```

Comment:
This is quite impressive! I just wanted to share a method in which you could implement drag and drop for each element using these events:
```js
// draggable elements
(() => {
  let element;
  /* events fired on the draggable target */
  document.addEventListener('drag', function (event) {
  });

  document.addEventListener('dragstart', function (event) {
    // store a ref. on the dragged elem
    element = event.target;
    // make it half transparent
    event.target.style.opacity = 0.5;
  });

  document.addEventListener('dragend', function (event) {
    // reset the transparency
    event.target.style.opacity = '';
  });

  /* events fired on the drop targets */
  document.addEventListener('dragover', function (event) {
    // prevent default to allow drop
    event.preventDefault();
  });

  document.addEventListener('dragenter', function (event) {
    // highlight potential drop target when the draggable element enters it
    event.target.style.background = 'purple';
  });

  document.addEventListener('dragleave', function (event) {
    // reset background of potential drop target when the draggable element leaves it
    event.target.style.background = '';
  });

  document.addEventListener('drop', function (event) {
    // prevent default action (open as link for some elements)
    event.preventDefault();
    // move dragged elem to the selected drop target
    event.target.style.background = '';
    element.parentNode.removeChild(element);
    event.target.appendChild(element);
  });
})();
```